### DISCUSSION TITLE
Toggleterm

### DISCUSSION THREAD

**Discussion by crivotz** [folke/edgy.nvim#11] [Q&A] [answered]
Hi, I use toggleterm with some special configurations to open some other programs from CLI in a float window and normal terminal in a bottom bar.
I set edgy in this way and works perfectly for terminal normal bar:
```
{
  ft = "toggleterm",
  title = "TERMINAL",
  size = { height = 0.4 },
},
```
Is there a way to recognize other CLI commands and left in a float?

`local lazygit = Terminal:new({ cmd = "lazygit", hidden = true, direction = "float" })`

Thanks for all the plugins

**Comment by folke** [comment]
See here https://github.com/folke/edgy.nvim/issues/10

**Comment by folke** [comment, marked as answer]
I just updated the readme and changed the toggleterm example to exclude floating windows

**Reply by kevintraver** [reply]
Is there a way to filter/exclude when _toggleterm_ is opened in a new **_tab_**?

**Comment by crivotz** [comment]
Thanks ;)
